### PR TITLE
CLOC (Line Count Tool) Action

### PR DESCRIPTION
**Pull request by intere**
As part of our CI build process, we perform line counts.  CLOC is a tool that will create line count reports in a format that Jenkins supports.  This action allows you to generate them (with various configurations) and put them where you please.

### CODE REVIEW

File: lib/fastlane/actions/cloc.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      def self.description
+        "Generates a Code Count that can be read by Jenkins (xml format)"
+      end
+
+      def self.details
+        "This action will run cloc to generate a SLOC report that the Jenkins SLOCCount plugin can read.  See https://wiki.jenkins-ci.org/display/JENKINS/SLOCCount+Plugin for more information."
+      end
+
+      def self.available_options
+        # Define all options your action supports.
```
Comment (KrauseFx) [inline lib/fastlane/actions/cloc.rb]: Please remove these comments

File: docs/Actions.md [hunk 2]
```diff
@@ -1727,6 +1727,15 @@ This can be used to store some generated URL or value for easy copy & paste (e.g
 clipboard(value: lane_context[SharedValues::HOCKEY_DOWNLOAD_LINK])
 ```
 
+### cloc
+
+You can generate a Line Count (that is readable by Jenkins - see the SLOCCount Plugin for more details).  This action is a wrapper around the cloc tool: https://github.com/AlDanial/cloc
```
Comment (KrauseFx) [inline docs/Actions.md]: Can you move the link to the headline of `cloc` similar to what `Podio` does?

Comment (KrauseFx) [inline docs/Actions.md]: Can you do an inline-link to `SLOCCount Plugin` website?

**Comment by KrauseFx** [comment]
Hey @intere, thanks for submitting this PR, this looks really great! :+1: I added some comments before it's ready to be merged

**Comment by intere** [comment]
@KrauseFx - I'll have your review comments addressed shortly.  Thank you for the feedback and the quick turnaround on my PR.  It's great to be a part of the community.

**Comment by KrauseFx** [comment]
This is great, thank you so much! I haven't tried it myself, but the Ruby part looks good. One last thing, could you please squash your commits to one? :+1:

**Comment by intere** [comment]
I'm going to close this pull request and issue a new one with the single commit

**Comment by intere** [comment]
New Pull Request: https://github.com/fastlane/fastlane/pull/1081

**Pull request closed without merging**